lib/features/hover-fix/index.js

```diff
@@ -0,0 +1,14 @@
+import DraggingModule from '../dragging';
+
+import HoverFix from './HoverFix';
+
+
+export default {
+  __init__: [
+    'hoverFix'
+  ],
+  __depends__: [
+    DraggingModule
+  ],
+  hoverFix: [ 'type', HoverFix ],
+};
```

Comment:
Since we ship this as a standalone module, we have to integrate it into downstream libraries as `bpmn-js` separately, right?

Reply:
@azeghers We decided to not break existing behavior and keep it included per default, right?